index.js

```diff
@@ -1 +1,107 @@
-console.log('hello')
\ No newline at end of file
+let gameFlow = document.querySelector('.game')
+let area = document.getElementById("touch-area")
+let scoreElement = document.getElementById('score')
+let scoreCounter = 0 // for score
+
+const gameFlowHeight = parseInt(window.getComputedStyle(gameFlow).getPropertyValue('height'))
+
+const blockClasses = ['left-arrow', 'up-arrow', 'down-arrow', 'right-arrow']
+
+class Block {
+    id = Date.now
+    className = getRandom().itemClass
+
+    createItem() {
+        const newItem = document.createElement('div')
+        newItem.setAttribute('class', `${this.className} drop-block`)
+        newItem.setAttribute('id', `${this.id}`)
+        return newItem
+    }
+}
+
+
+const checkTouch = (item) => {
+    const itemPosition = parseInt(window.getComputedStyle(item).getPropertyValue('top'))
+    const excellentArea = itemPosition >= (gameFlowHeight - 100) + 10 && itemPosition <= gameFlowHeight - 60
+    const goodArea = itemPosition >= (gameFlowHeight - 100) - 10 && itemPosition <= gameFlowHeight - 40
+
+    if (excellentArea) {
+        scoreCounter += 2
+        scoreElement.textContent = scoreCounter
+        return
+    } else if (goodArea) {
+        console.log('correct')
+        scoreCounter += 1
+        scoreElement.textContent = scoreCounter
+        return
+    }
+
+}
+
+const getRandom = () => {
+    const elements = Math.floor((Math.random() * 20) + 5)
+    const seconds = Math.floor(Math.random() * 2000 + 200)
```

Comment:
Magic numbers should be put into the constants, so it's clear, what they do. Also, if ```2000``` is milliseconds, it doesn't match with variable name